controllers/activemqartemis_reconciler.go

```diff
@@ -2355,6 +2368,34 @@ func brokerPropertiesData(props []string) map[string]string {
 	return contents
 }
 
+func bpBrokerPropertiesData(customResource *brokerv1beta1.ActiveMQArtemis, client rtclient.Client) (map[string]map[string]string, error) {
+
+	propData := make(map[string]map[string]string)
+
+	for _, secretName := range customResource.Spec.DeploymentPlan.ExtraMounts.Secrets {
+		if strings.HasSuffix(secretName, brokerPropsSuffix) {
+
+			bpSecret := &corev1.Secret{}
+			bpSecretKey := types.NamespacedName{
+				Name:      secretName,
+				Namespace: customResource.Namespace,
+			}
+			if err := resources.Retrieve(bpSecretKey, client, bpSecret); err != nil {
+				return nil, err
+			}
+
+			secretData := make(map[string]string)
+
+			//broker-N.<prop-file-key> contains per broker configurations
+			for key, data := range bpSecret.Data {
+				secretData[key] = string(data)
```

Comment:
The data size could be very huge and it is not used, only the keys are required to mount indexed broker.properties files

Reply:
+1